docker-compose.yml

```diff
@@ -5,15 +5,19 @@ web:
     - '443/tcp'
   volumes:
     - './web:/var/www/html'
+    - './drupal.make:/var/www/html/drupal.make'
     - './docker/settings.php:/var/www/html/sites/default/settings.php:ro'
+    - './docker/settings.local.php:/var/www/html/sites/default/settings.local.php:ro'
     - './docker/drupal-cron.conf:/etc/cron.d/drupal-cron.conf:ro'
     - './docker/prepare-environment.sh:/etc/my_init.d/prepare-environment.sh'
     - './docker/php.ini:/etc/php/5.6/mods-available/ding.ini'
+    - "${SSH_AUTH_SOCK}:/tmp/ssh-agent"
```

Comment:
This approach is not quite good enough. It only works on Linux, and requires that the container is built while key forwarding is enabled and active. [This thread](https://www.flowdock.com/app/reload/docker/threads/Oq9XvxEypJhWV8H-BX2KgLx8xi-) has alternative suggestions for a solution.